.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -841,7 +841,7 @@ jobs:
           NEXUS_PASSWORD: ${{ secrets.sdk-ossrh-password }}
         with:
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: ":release${{ inputs.release-profile }} -PpublishingPackageGroup=com.swirlds --scan -PpublishSigningEnabled=true --no-configuration-cache"
+          arguments: "release${{ inputs.release-profile }} -PpublishingPackageGroup=com.swirlds --scan -PpublishSigningEnabled=true --no-configuration-cache"
```

Comment:
For my own learning, what does the : do?

Reply:
`:releaseSomething` invokes the task `releaseSomething` **only**  in the root project and fails if it is not there. That's the problem we had before this PR for tasks that do not exist in the root.

`releaseSomething` invokes multiple tasks called `releaseSomething` – in all projects where it exists. It skips projects that do not have it (e.g. the root) without error. 

I have a  section in a video that explains this a bit more: https://www.youtube.com/watch?v=sOo0p4Gpjcc&list=PLWQK2ZdV4Yl2k2OmC_gsjDpdIBTN0qqkE

With the older solution, we did `:releaseSomething` and then controlled which tasks are started in addition through dependency from that task.
With the new solution, we allow `releaseSomething` again for all cases. It might lead to more tasks starting than we need, but will skip the ones we do not want via the deactivation logic.

Usually I prefer the `:` solution because it is exact. But in this case, it is complex to get it right with all the different publishing cases.